src/main.cc

```diff
@@ -798,6 +798,20 @@ int main(int argc, char* argv[])
                    { "clear", { false, "clear dead sessions" } },
                    { "ro", { false, "readonly mode" } } }
     };
+
+    if (argc == 2 and String(argv[1]) == "--help")
```

Comment:
Use the parameter parser here, and `-help` instead of `--help`, we dont use the long option syntax anywhere.